src/test/kotlin/com/terraformation/backend/nursery/db/batchStore/BatchStoreWithdrawTest.kt

```diff
@@ -688,6 +705,189 @@ internal class BatchStoreWithdrawTest : BatchStoreTest() {
+                      speciesId = speciesId,
+                      germinatingQuantity = 1 + 4,
+                      notReadyQuantity = 2 + 5,
+                      readyQuantity = 3 + 6,
+                      latestObservedGerminatingQuantity = 1,
+                      latestObservedNotReadyQuantity = 2,
+                      latestObservedReadyQuantity = 3,
+                      version = 2),
+              ),
+              batchesDao.fetchByFacilityId(destinationFacilityId),
+              "Should have created one new batch")
+        },
+        {
+          assertEquals(
+              listOf(
+                  BatchWithdrawalsRow(
+                      batchId = species1Batch1Id,
+                      destinationBatchId = newBatch.id,
+                      germinatingQuantityWithdrawn = 1,
+                      notReadyQuantityWithdrawn = 2,
+                      readyQuantityWithdrawn = 3,
+                      withdrawalId = firstWithdrawal.id),
+                  BatchWithdrawalsRow(
+                      batchId = species1Batch1Id,
+                      destinationBatchId = newBatch.id,
+                      germinatingQuantityWithdrawn = 4,
+                      notReadyQuantityWithdrawn = 5,
+                      readyQuantityWithdrawn = 6,
+                      withdrawalId = secondWithdrawal.id),
+              ),
+              batchWithdrawalsDao.findAll().sortedBy { it.germinatingQuantityWithdrawn },
+              "Should have created batch withdrawals")
+        },
+        {
+          assertEquals(
+              listOf(
+                  BatchQuantityHistoryRow(
+                      batchId = newBatch.id!!,
+                      createdBy = user.userId,
+                      createdTime = firstWithdrawalTime,
+                      historyTypeId = BatchQuantityHistoryType.Observed,
+                      germinatingQuantity = 1,
+                      notReadyQuantity = 2,
+                      readyQuantity = 3),
+                  BatchQuantityHistoryRow(
+                      batchId = newBatch.id!!,
+                      createdBy = user.userId,
+                      createdTime = secondWithdrawalTime,
+                      historyTypeId = BatchQuantityHistoryType.Computed,
+                      germinatingQuantity = 1 + 4,
+                      notReadyQuantity = 2 + 5,
+                      readyQuantity = 3 + 6),
+                  BatchQuantityHistoryRow(
+                      batchId = species1Batch1Id,
+                      createdBy = user.userId,
+                      createdTime = secondWithdrawalTime,
+                      historyTypeId = BatchQuantityHistoryType.Computed,
+                      germinatingQuantity = 10 - 1 - 4,
+                      notReadyQuantity = 20 - 2 - 5,
+                      readyQuantity = 30 - 3 - 6,
+                      withdrawalId = secondWithdrawal.id),
+                  BatchQuantityHistoryRow(
```

Comment:
insert above previous one just for readable order?

Reply:
> insert above previous one just for readable order?

I did this because the assertion orders these by ready quantity, but you're right, it'd be clearer the other way and I can  just make it use a set instead of a list.